packages/amplify-cli/src/extensions/amplify-helpers/resource-status.ts

```diff
@@ -179,6 +180,9 @@ async function getResourcesToBeUpdated(amplifyMeta, currentAmplifyMeta, category
   await asyncForEach(Object.keys(amplifyMeta), async categoryName => {
     const categoryItem = amplifyMeta[categoryName];
     await asyncForEach(Object.keys(categoryItem), async resource => {
+      if (categoryName === 'analytics') {
```

Comment:
This is not right. Removal should happen on push as we discussed? an `amplify status` should not change resources.

Reply:
Discussed offline